src/Workspaces/CSharp/Portable/CodeGeneration/MethodGenerator.cs

```diff
@@ -206,7 +208,28 @@ internal static class MethodGenerator
         {
             return !method.ExplicitInterfaceImplementations.Any() && !method.IsOverride
                 ? method.TypeParameters.GenerateConstraintClauses()
-                : default;
+                : GenerateDefaultConstraints(method);
```

Comment:
Are there any cases where this will be generating extra `default` constraints?

Reply:
i think this is fine.